proto/eth/service/key_management.proto

```diff
@@ -133,3 +154,54 @@ message DeletedKeystoreStatus {
   Status status = 1;
   string message = 2;
 }
+
+
+message ListRemoteKeysResponse {
+  message Keystore {
+    bytes pubkey = 1;
+    string url = 2;
+    bool readonly = 3;
+  }
+  repeated Keystore data = 1;
+}
+
+message ImportRemoteKeysRequest {
+  message Keystore {
+    bytes pubkey = 1;
+    string url = 2;
+  }
+  repeated Keystore remote_keys = 1;
+}
+
+message ImportRemoteKeysResponse {
+  repeated ImportedRemoteKeysStatus data = 1;
+}
+
+message DeleteRemoteKeysRequest {
+  repeated bytes pubkeys = 1;
+}
+
+message DeleteRemoteKeysResponse {
+  repeated DeletedRemoteKeysStatus data = 1;
+}
+
+message ImportedRemoteKeysStatus {
+  enum Status {
+    IMPORTED = 0;
```

Comment:
It's generally good to reserve 0 for `UNKNOWN` since Go defaults int type to 0. In the event where marshal/unmarshal fails, your remoteKeyStatus.status would default to `IMPORTED`, and that's dangerous

Reply:
`NOT_FROUND` also makes sense for 0 as well

Reply:
oh i didn't realize this will keep that in mind and change here